examples/apps/djapp/legacy/4_create_initial_mesh_components/virtual-services.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: appmesh.k8s.aws/v1beta1
+apiVersion: appmesh.k8s.aws/v1beta2
 kind: VirtualService
 metadata:
   name: jazz.prod.svc.cluster.local
```

Comment:
`VirtualRouter` is no longer part of `VirtualService`

Reply:
Same issue, fixed.